app/views/hotspots/_form.html.erb

```diff
@@ -1,21 +1,29 @@
-<%= form_for @hotspot do |f| %>
-  <div class="form-group">
-    <%= f.text_field :title, placeholder: 'Title', class: 'form-control' %>
+<div class="container">
+  <div class="row">
+    <div class="col-xs-0"></div>
+<div id = 'hotspots_form' class="col-xs-5">
```

Comment:
This should be indented in in to match the indentation on line 3, since this inside the div on line 2.